evaluate.js

```diff
@@ -0,0 +1,95 @@
+const operators = {
+    "^": {
+        precedence: 4,
+        associativity: "Right"
```

Comment:
move Right and Left to consts if you use them sevaral time
as well as codes for precedence